docs/jax.numpy.rst

```diff
@@ -29,6 +29,14 @@ input data.
 
 Not every function in NumPy is implemented; contributions are welcome!
 
+.. Generate the list below as follows:
+   >>> import jax.numpy, numpy
+   >>> fns = set(dir(numpy)) & set(dir(jax.numpy)) - set(jax.numpy._NOT_IMPLEMENTED)
+   >>> print('\n'.join('    ' + x for x in fns if callable(getattr(jax.numpy, x))))
```

Comment:
Add `# doctest: +SKIP` at the end of this line.

Reply:
Fixed, thanks!